### DISCUSSION TITLE
Importing Racks and Encountering error

### DISCUSSION THREAD

**Discussion by serveitup** [netbox-community/netbox#11669] [Q&A]
I'm trying to import racks from a local host that I was testing the app out on, to a deployed version. 
When I copy and paste the CSV or Import the file directly, I'm getting this issue. 

```
<class 'dcim.models.racks.Rack.site.RelatedObjectDoesNotExist'>

Rack has no site.
```

How do I solve this?

Edit: Trying to create a site first.
I was thinking for some that it meant no site in my config.py.

**Comment by candlerb** [comment]
Can you show your CSV data? The header line plus one line of data.

It seems that most likely the "site" column is either missing, or is referring to a non-existent site (as in "Organization > Sites" in the GUI - nothing to do with config.py)
